CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to BaseApp
+
+Thank you for your interest in contributing to BaseApp! We welcome contributions from the community to help improve and grow the project. Please take a moment to review these guidelines before getting started.
+
+## Table of Contents
+
+- [Reporting Bugs](#reporting-bugs)
+- [Requesting Features](#requesting-features)
+- [Contributing Code](#contributing-code)
+- [Code of Conduct](#code-of-conduct)
+
+---
+
+## Reporting Bugs
+
+When you encounter a bug or unexpected behavior in BaseApp, please follow these steps to report it:
+
+1. Check the [GitHub Issues](https://github.com/silverlogic/baseapp-frontend/issues) to ensure that the bug hasn't already been reported.
+
+2. If the issue hasn't been reported, create a new issue by clicking on the "New Issue" button.
```

Comment:
let's make sure people use the right label too, maybe for bugs requests we could use the [bug](https://github.com/silverlogic/baseapp-frontend/labels/bug) label